.github/workflows/qat.yml

```diff
@@ -1,89 +1,75 @@
-name: QAT
+name: 'Checkly QAT'
 
 on:
   workflow_call:
     inputs:
       QA_SITE_DOMAIN:
         default: ''
         description: 'QA Site Domain'
-        required: false
+        required: true
```

Comment:
This can't be changed until every theme (and `pmc-plugins`) is updated to pass this value.